src/components/Menu-new/Menu.tsx

```diff
@@ -0,0 +1,112 @@
+import { ReactElement, useCallback, useState } from 'react';
+import { Icon, View, Text } from '@aws-amplify/ui-react';
+import Link from 'next/link';
+import { directory } from './buildDirectory.mjs';
+import styles from './Menu.module.scss';
+import { useRouter } from 'next/router';
+
+export function Menu(): ReactElement {
+  const rootPage = directory;
+  const platformOverviewPage = rootPage.children[0];
+
+  return (
+    <nav className={styles['menu']}>
+      <View marginBottom="16px">
```

Comment:
A reminder to convert these style props into classes when you get to it and that we can use Amplify UI css custom properties (I know the styles/designs aren't finalized, that can all be cleaned up in a separate PR).